hr_timesheet_print_employee_timesheet/wizard/hr_analytical_timesheet_employees.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import collections
+import datetime
+import babel.dates
+from dateutil.relativedelta import relativedelta
+from openerp import models, fields, api
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class HrAnalyticalTimesheetEmployees(models.TransientModel):
+    _name = 'hr.analytical.timesheet.employees'
+    _description = 'Print timesheets per employee'
+
+    employee_ids = fields.Many2many(
+        'hr.employee', string='Employees',
+        default=lambda self: self.env.context.get('active_ids', []),
+        required=True)
+    date_start = fields.Date(
```

Comment:
@hbrunn Can you please add a constrain about dates, to not be able to select date start bigger than date end.

Reply:
@feketemihai can you re-review?